ntbs-service/DataAccess/ReferenceDataRepository.cs

```diff
@@ -183,17 +183,22 @@ public async Task<List<string>> GetPhecCodesMatchingRolesAsync(IEnumerable<strin
 
         public IQueryable<TBService> GetTbServicesQueryable()
         {
-            return _context.TbService;
+            return _context.TbService.OrderBy(s => s.Name);
         }
 
         public IQueryable<TBService> GetDefaultTbServicesForNhsUserQueryable(IEnumerable<string> roles)
         {
-            return _context.TbService.Where(tb => roles.Contains(tb.ServiceAdGroup));
+            return _context.TbService
+                .Where(tb => roles.Contains(tb.ServiceAdGroup))
```

Comment:
Judging from this https://stackoverflow.com/questions/16746389/orderby-clause-before-where-clause-performance the order of where and orderby means we might be able to use the GetTbServicesQueryable here with the ordering included but I haven't fully convinced myself of it yet